src/utils/interfaces.ts

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-import { ClickMode, HoverMode, InteractivityDetect, MoveDirection, OutMode, ShapeType } from "./enums";
+import { ClickMode, HoverMode, InteractivityDetect, MoveDirection, OutMode, ShapeType, ProcessBubbleType } from "./enums";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5012291)